instance/testing.py

```diff
@@ -8,12 +8,12 @@
 UNKNOWN_FILE_THUMBNAIL = 'unknown.jpeg'
 
 #: S3 Configuration; AWS credentials with restricted access
-AWS_ACCESS_KEY = environ.get('AWS_ACCESS_KEY')
-AWS_SECRET_KEY = environ.get('AWS_SECRET_KEY')
-AWS_BUCKET = environ.get('AWS_BUCKET', 'imgee-testing')
+#: S3 Configuration
+AWS_ACCESS_KEY = 'AKIAIRIRVOH3RE5SKR4A'
+AWS_SECRET_KEY = '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'
```

Comment:
should this be public?

Reply:
They've been revoked. committed by mistake.